seatunnel-server/seatunnel-app/src/main/java/org/apache/seatunnel/app/controller/ScriptController.java

```diff
@@ -0,0 +1,106 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.app.controller;
+
+import org.apache.seatunnel.app.common.Result;
+import org.apache.seatunnel.app.domain.request.script.AddEmptyScriptReq;
+import org.apache.seatunnel.app.domain.request.script.ScriptListReq;
+import org.apache.seatunnel.app.domain.request.script.UpdateScriptContentReq;
+import org.apache.seatunnel.app.domain.request.script.UpdateScriptParamReq;
+import org.apache.seatunnel.app.domain.response.script.ScriptParamRes;
+import org.apache.seatunnel.app.domain.response.script.ScriptSimpleInfoRes;
+import org.apache.seatunnel.app.service.IScriptService;
+
+import io.swagger.annotations.ApiImplicitParam;
+import io.swagger.annotations.ApiImplicitParams;
+import io.swagger.annotations.ApiOperation;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.annotation.Resource;
+import javax.validation.constraints.NotNull;
+
+import java.util.List;
+
+@RequestMapping("/api/v1/script")
+@RestController
+public class ScriptController {
+    @Resource
+    private IScriptService iScriptService;
+
+    @PostMapping("/addEmptyScript")
+    @ApiOperation(value = "add an empty script", httpMethod = "POST")
+    public Result<Void> addEmptyScript(@RequestBody @NotNull AddEmptyScriptReq addEmptyScriptReq) {
+        iScriptService.addEmptyScript(addEmptyScriptReq);
+        return Result.success();
+    }
+
+    @PutMapping("/updateScriptContent")
+    @ApiOperation(value = "update script", httpMethod = "POST")
+    public Result<Void> updateScriptContent(@RequestBody @NotNull UpdateScriptContentReq updateScriptContentReq) {
+        iScriptService.updateScriptContent(updateScriptContentReq);
+        return Result.success();
+    }
+
+    @DeleteMapping("/delete")
+    @ApiOperation(value = "delete script", httpMethod = "POST")
+    @ApiImplicitParams({
+            @ApiImplicitParam(name = "id", value = "script id", dataType = "Integer"),
+    })
+    public Result<Void> delete(@RequestParam @NotNull Integer id) {
+        iScriptService.delete(id);
+        return Result.success();
+    }
+
+    @PostMapping("/list")
+    @ApiOperation(value = "script list", httpMethod = "POST")
+    public Result<List<ScriptSimpleInfoRes>> list(@RequestBody @NotNull ScriptListReq scriptListReq) {
+        return Result.success(iScriptService.list(scriptListReq));
```

Comment:
We use the standard Restful style, so the operation methods of add and delete will not need to be reflected in the Url

Reply:
OK, so I'm going to remove all the `verbs`